kernel/src/transaction/mod.rs

```diff
@@ -91,6 +74,17 @@ impl Transaction {
             .table_configuration()
             .ensure_write_supported()?;
 
+        let add_files_schema = Arc::new(StructType::new(vec![
+            StructField::not_null("path", DataType::STRING),
+            StructField::not_null(
+                "partitionValues",
+                MapType::new(DataType::STRING, DataType::STRING, true),
+            ),
+            StructField::not_null("size", DataType::LONG),
+            StructField::not_null("modificationTime", DataType::LONG),
+            StructField::not_null("dataChange", DataType::BOOLEAN),
+        ]));
+
```

Comment:
Why do we 'manually' have to create `add_files_schema` here? Can't we use `Add::to_schema()`?